lgc/builder/ImageBuilder.cpp

```diff
@@ -1831,3 +1836,36 @@ Value *ImageBuilder::handleFragCoordViewIndex(Value *coord, unsigned flags, unsi
 
   return coord;
 }
+
+// =====================================================================================================================
+// Handle the uniform index by track back through a descriptor load and create a readfirstlane for it
+//
+// @param imageSampleInst : The instruction to get the image and sampler descriptors
+// @param operandIdx :
+// @param instName : Name to give instruction(s)
+void ImageBuilder::handleUniformIndex(Instruction *imageSampleInst, ArrayRef<unsigned> operandIdxs,
+                                      const Twine &instName) {
```

Comment:
"Handle" is a vague term, replacing it with a more concrete term would be better. My suggestion is `enforceUniformIndex`, which can be explained as:
```
// Enforce that the descriptors loaded for use by @p imageSampleInst are loaded using a uniform index
// by tracking back through a descriptor load to find the underlying index and creating a readfirstlane for it.
```
```